app/channel.go

```diff
@@ -1479,7 +1475,13 @@ func (a *App) LeaveChannel(channelId string, userId string) *model.AppError {
 		uc <- store.StoreResult{Data: user, Err: err}
 		close(uc)
 	}()
-	ccm := a.Srv.Store.Channel().GetMemberCount(channelId, false)
+
+	cc := make(chan store.StoreResult, 1)
+	go func() {
```

Comment:
Why cc instead of ccm?

Reply:
cc = countChan
may be better mcc = memberCountChan ?
I did not understand ccm, is it "chan for counting members" or "count channel members"?

Reply:
Yes, for sure wasn't very consistent, I like the `mcc` option.